docs/source/concept_models/vivarium_ciff_sam/concept_model.rst

```diff
@@ -563,6 +572,12 @@ Final outputs to report in manuscript
 7.0 Limitations
 +++++++++++++++
 
+See project technical report.
```

Comment:
Might be helpful to link to report here: I:\RTs_and_Projects\Sim_Science_RT\04_Projects\Archive\Severe Acute Malnutrition Sim (CIFF)\Deliverables\20220311_IHME simulation report.pdf

Reply:
Added, thank you!